src/client/routes/Portfolio/index.tsx

```diff
@@ -92,11 +98,37 @@ export const Portfolio = () => {
 
   const availableRoles = [BORROWER_POSITION_ROLE, LENDER_POSITION_ROLE];
 
+  // TODO: Move this from market and portfolio pages into a shared location
+  const defaultLineCategories: UseCreditLinesParams = {
+    // using i18m translation as keys for easy display
+    'market:featured.highest-credit': {
+      first: 3,
+      // NOTE: terrible proxy for total credit (oldest = most). Currently getLines only allows filtering by line metadata not modules'
+      orderBy: 'start',
+      orderDirection: 'asc',
+    },
+    'market:featured.highest-revenue': {
+      first: 16,
+      // NOTE: terrible proxy for total revenue earned (highest % = highest notional). Currently getLines only allows filtering by line metadata not modules'
+      orderBy: 'defaultSplit',
+      orderDirection: 'desc',
+    },
+    'market:featured.newest': {
+      first: 15,
+      orderBy: 'start', // NOTE: theoretically gets lines that start in the future, will have to refine query
+      orderDirection: 'desc',
+    },
+  };
+
   useEffect(() => {
     if (!isValidAddress(portfolioAddress!)) {
       dispatch(AlertsActions.openAlert({ message: 'Please connect wallet.', type: 'error' }));
     } else if (portfolioAddress && isValidAddress(portfolioAddress)) {
       dispatch(LinesActions.getUserPortfolio({ user: portfolioAddress.toLocaleLowerCase() }));
+      // TODO: only dispatch if userPortfolio doesn't provide what we need to populate linesMap
```

Comment:
is this where the infinite loop or whatever the useEffect issue was?

Reply:
No, not an issue with `useEffect`. We are dispatching `getLines` when user comes to portfolio page because the `getUserPortfolio` query doesn't retrieve all the lines for which the user is a lender.